test/passing/ite.ml

```diff
@@ -41,3 +41,64 @@ f
   else () )
 
 let () = if [@test] true then () else if [@other] true then ()
+
+[@@@ocamlformat "if-then-else=fit-or-vertical"]
```

Comment:
I think it would be better to have a `ite_fit_or_vertical.ml` symlink to this file and `ite_fit_or_vertical.ml.opts` that add the option. Same for `keyword-first`. That way, future tests will be run with the 3 options.

Reply:
@jberdine what is your opinion on introducing symbolic links for the tests?

Reply:
I did that in https://github.com/ocaml-ppx/ocamlformat/pull/619

Reply:
Symlinks seem ok to me.